src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -88,15 +87,62 @@
 
 @postgres_dialect.segment(replace=True)
 class FunctionDefinitionGrammar(BaseSegment):
-    """This is the body of a `CREATE FUNCTION AS` statement."""
+    """This is the body of a `CREATE FUNCTION AS` statement.
+
+    Options supported as defined in https://www.postgresql.org/docs/9.1/sql-createfunction.html
```

Comment:
Looks to me like you've included TRANSFORM which was added since 9.1 so should we update this to 9.5?

```suggestion
    Options supported as defined in https://www.postgresql.org/docs/9.5/sql-createfunction.html
```

Reply:
Sorted, I've also brought it forward to 13, and added in support for PARALLEL and SUPPORT